ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-bgp-evpn-vpn-import-pruning.cfg

```diff
@@ -62,6 +62,8 @@ router bgp 65101
       neighbor EVPN-OVERLAY-PEERS activate
       no neighbor MLAG-IPv4-UNDERLAY-PEER activate
       route import match-failure action discard
+      route import ethernet-segment ip mass-withdraw
+      route export ethernet-segment ip mass-withdraw
```

Comment:
These changes do not match your yaml changes. Please rerun molecule on your side and commit the changes.

Reply:
I executed molecule again. Now these lines are not present in documents and configs.